### DISCUSSION TITLE
Unable to locate publishable resources

### DISCUSSION THREAD

**Discussion by radiusmediaaus** [spatie/laravel-backup#1036] [Q&A] [answered]
Running the latest Laravel version.

Installed the backup package, however when running this command
php artisan vendor:publish --provider="Spatie\Backup\BackupServiceProvider"   

It returns the following;
Unable to locate publishable resources.
Publishing complete.

But no backup config file has been created under the config folder.

Please advise what might be going on here.

**Comment by radiusmediaaus** [comment]
Surely someone can answer my question?

**Comment by mehluli92** [comment]
Actually having the same challenge

**Comment by taghwo** [comment, marked as answer]
Check your vendors directory if spatie/laravel-backup is there. Composer timed out when i was installing the package so i got same error when publishing. Composer update set things up for me.

**Comment by Myokhing** [comment]
Thankyou for your answer
